Cargo.toml

```diff
@@ -35,17 +36,20 @@ serde_with = "2.2"
 strum = "0.24"
 strum_macros = "0.24"
 systemd-zbus = "0.1"
-# ToDo: set to 0.3 once we support rust 1.63
-time = { version = "=0.3.19", features = ["formatting"] }
+time = { version = "=0.3.23", features = ["formatting"] }
```

Comment:
Do we still need explicitly this version? Why so?

Reply:
we need was is supported by 1.65 
afaik 0.3.24 requires 1.67